README.md

```diff
@@ -99,32 +99,69 @@ There are several ways to help. Publishing a bug or a feature request is a good
 
 **Pull requests** are of course also very welcome. Please make sure that you're following the [angular commit guidelines](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#commits) and to also include the issue number in your commit message, if you're fixing a particular issue (e.g.: `feat: add nice feature with the number #31`).
 
-### Running the development server
-To run the development server you need to have node installed at least in the version 10. Go to https://nodejs.org for installation instructions.
-
-**Clone repo**
-```
-git clone https://github.com/johannesjo/super-productivity.git
-```
-**Install dependencies**
-```
-cd super-productivity
-npm install
-npm install -g @angular/cli
-```
-**Run the dev server**
-```
-ng serve
-```
-Afterwards you can open http://localhost:4200 in your browser. For most adjustments this should be enough and also enables you to use the [redux dev tools](https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd?hl=de). 
-
-If you want to run the app standalone do the following:
+### How to contribute 
```

Comment:
To be honest I find the the instructions a bit to detailed for the readme. Normally a developer with the required experience to work with the technologies used in the project should be more than fine with the instructions provided. But maybe we can use this to the wiki and provide a link from there for windows users: https://github.com/johannesjo/super-productivity/wiki

Reply:
To your point if people already have  the skills they will be able to extrapolate the instructions. Moving to wiki is fine. The instructions are not really Windows specific. They map to Linux and Mac. I'm not sure where you saw Windows specific instructions. I will re-read it to make sure to make it more generic.

Reply:
Let's move them to the wiki then! I think the readme is more meant to give an quick overview over everything. I like the instructions. They're pretty helpful to everyone that never worked with those technologies. So it would be cool to have them available.